### PR TITLE
Portfolio-finish-mobile-version

### PR DESCRIPTION
**Pull request by djo1975**
I add html  index and css styles files and all needed files. You can follow the index.html it.

### CODE REVIEW

**Review by Tufahel — CHANGES_REQUESTED** [review]
Hi @djo1975, :wave: 
 
While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
 
## Invalid Code Review Request :red_circle: 
You have submitted a project, that has no pull request description. Kindly include one and resubmit. If you don't have an idea of how to include a pull request description, you can read this [article](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request).
 
#### PS: Concentrate on point of the article. 
Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready.
 
Cheers and Happy coding! :clap: :clap: 
 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by djo1975** [comment]
did I make the description of this time

**Comment by djo1975** [comment]
did I make the description of this time

**Comment by TuyishimireEric** [comment]
# Hi @djo1975, :wave:

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
## Invalid Code Review Request :red_circle:

- [ ] You have submitted a project, that has no pull request description. Kindly Remember the Pull Request should have a short summary describing introduced changes in more detail. :joy: 
    - kindly describe introduced changes instead of having
    image


Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready.

### Cheers and Happy coding! :clap: :clap:
---
Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

_Invalid Code Review Request does not count into the code reviews limit._

**Comment by Tufahel** [comment]
Hi @djo1975 :wave: 
Screenshot from 2022-12-03 22-23-35
- I can see that you have added a description now. Kindly add a summary `what have you implemented in this project` in the description.
- Then ask for another review from your `Microverse Dashboard`. You have not lost any review for this.

**Comment by djo1975** [comment]
sorry I can find out how to edit
please explain me what I have to do

File: index.html [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+                <div class="items">
+                  <div class="portofolio-img-2"></div>
+                  <h2 class="project-title">Multi-Post <br> Stories</h2>
+                    <div class="c-b-2015">
+                    <h6>CANOPY</h6>
+                    <ul class="c-b-2015-ul">
+                      <li>Back End Dev</li>
+                      <li>2015</li>
+                    </ul>
+                  </div>
```
Comment (mckent05) [inline index.html:103]: - Good job aligning the content of your project card with the `flex` display property, but there are some differences that still need to be addressed. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your project:
-- The `see project` button should be below the project description text, please fix this for all your project cards 👍 👍 
-- The list of technologies should also be below the project description text, please fix this for all your project cards 🙏 🙏 
wgawrgaw

Figma Design:
wgawrgaw

File: index.html [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+                      <li><img src="css.svg">  CSS </li>
+                    </section>
+        <section class="other">
+              <li class="other-item">Frameworks</li>
+              <li class="others-item-2"><img src="jsc.svg" alt="Right Arrow"></li>
+            </section>
+            <section class="others-section-ii">
+              <li class="others-item-3">Skills</li>
+              <li class="others-arrow"><img src="jsc.svg" alt="Right Arrow"></li>
+            </section>
```
Comment (mckent05) [inline index.html:207]: - Good job on the work done so far, but there are still some differences that need to be addressed. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your project:
-- This section has no background image, but you used added some background images on your page, please correct this 👍 👍 
--The `Get my Resume` button doesn't match the design in the Figma template, please fix this by giving it some more width 🙏 🙏 
-- The following sections are missing in your design, I have highlighted them in the Figma template attached below
    --- The list of technologies under the languages section, please fix this 🙏 
rdbfb

Figma Design:
-- Please pay attention to all marked sections and include them in your design
utukk

File: index.html [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+        
+                <div class="flex-item">
+                  <ul class="rest">
+                      <li class="arrow"><img src="jsc.svg" alt="Down Arrow"></li>
+                    </ul>
+                    <section class="languages">
+                      <li><img src="js.svg"> JavaScript</li>
+                      <li><img src="html.svg"> HTML </li>
+                      <li><img src="css.svg">  CSS </li>
+                    </section>
```
Comment (mckent05) [inline index.html:199]: - For semantic reasons, when using the `li` tag, it is necessary that they are wrapped directly under a `ul` tag, please remove the `section` tag used in this part of your code and make sure your `li` tags are directly grouped under a `ul` tag. Please fix this 👍 👍

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @djo1975,

# STATUS: CHANGES REQUIRED ♻️ ♻️  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ 
- You have used the right workflow for this milestone 👍 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
**OPTIONAL** - You have a huge X-scroll on your page, for a better UI and User experience, I suggest you make sure you remove the x-scroll present on your page. Please fix this 👍 👍 

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**